extended-statistics.md

```diff
@@ -0,0 +1,163 @@
+- `column_name` specifies the column group, which might have multiple columns. Currently, you can only specify two column names.
+
+<details>
+<summary> How it works</summary>
+
+To improve access performance, each TiDB node maintains a cache in the system table `mysql.stats_extended` for extended statistics. After you register the extended statistics, the next time the `ANALYZE` statement is executed, TiDB will collect the extended statistics if the system table `mysql.stats_extended` has the corresponding objects.
+
+Each row in the `mysql.stats_extended` table has a `version` column. Once a row is updated, the value of `version` is increased. In this way, TiDB loads the table into memory incrementally, instead of fully.
+
+TiDB loads `mysql.stats_extended` periodically to ensure that the cache is kept the same as the data in the table.
+
+> **Warning:**
+>
+> It is **NOT RECOMMENDED** to directly operate on the `mysql.stats_extended` system table. Otherwise, inconsistent caches occur on different TiDB nodes.
+>
+> If you have mistakenly operated on the table, you can execute the following statement on each TiDB node. Then the current cache will be cleared and the `mysql.stats_extended` table will be fully reloaded:
+>
+> ```sql
+> ADMIN RELOAD STATS_EXTENDED;
+> ```
+
+</details>
+
+### Delete extended statistics
+
+To delete an extended statistics object, use the following statement:
+
+```sql
+ALTER TABLE table_name DROP STATS_EXTENDED stats_name;
+```
+
+<details>
+<summary>How it works</summary>
+
+After you execute the statement, TiDB marks the value of the corresponding object in `mysql.stats_extended`'s column `status` to `2`, instead of deleting the object directly.
+
+Other TiDB nodes will read this change and delete the object in their memory cache. The background garbage collection will delete the object eventually.
+
+> **Warning:**
+>
+> It is **NOT RECOMMENDED** to directly operate on the `mysql.stats_extended` system table. Otherwise, inconsistent caches occur on different TiDB nodes.
+>
+> If you have mistakenly operated on the table, you can use the following statement on each TiDB node. Then the current cache will be cleared and the `mysql.stats_extended` table will be fully reloaded:
+>
+> ```sql
+> ADMIN RELOAD STATS_EXTENDED;
+> ```
+
+</details>
+
+### Export and import extended statistics
+
+The way of exporting or importing extended statistics is the same as exporting or importing regular statistics. See [Introduction to Statistics - Import and export statistics](/statistics.md#import-and-export-statistics) for details.
+
+## Usage examples for correlation-type extended statistics
+
+Currently, TiDB only supports the correlation-type extended statistics. This type is used to estimate the number of rows in the range query and improve index selection. The following example shows how the correlation-type extended statistics are used to estimate the number of rows in a range query.
+
+### Stage 1. Define the table
```

Comment:
Is there a reason for using "Stage" instead of "Step"?
```suggestion
### Step 1. Define the table
```

Reply:
Looks like "Step" is a better word.